packages/qdrant-loader/src/qdrant_loader/core/file_conversion/file_converter.py

```
@@ -192,6 +178,20 @@ def _get_markitdown(self):
+    def _resolve_chat_model(self) -> str:
+        """Resolve chat model from unified LLM settings for MarkItDown captions."""
+        try:
+            from importlib import import_module
+
+            cfg_mod = import_module("qdrant_loader.config")
+            settings = cfg_mod.get_settings()
+            model = (settings.llm_settings.models or {}).get("chat")
+            if isinstance(model, str) and model.strip():
+                return model.strip()
+        except Exception:
+            pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)